### PR TITLE
ELPP-2769 Add quotes and refactor pull quotes

### PR DESCRIPTION
**Pull request by thewilkybarkid**
This replaces paragraphs in author responses being wrapped in `<i>`.

### CODE REVIEW

File: assets/sass/patterns/molecules/pull-quote.scss [hunk 1]
```diff
@@ -2,21 +2,16 @@
 
 .pull-quote {
   clear: both;
-  background: url("../img/icons/quote-start.svg") 0 34px no-repeat, linear-gradient(transparent, transparent);
+  // 24 is height of the image
+  background: url("../img/icons/quote-start.svg") 0 #{$blg-space-large-in-px - 24 - 14}px no-repeat, linear-gradient(transparent, transparent);
```
Comment (thewilkybarkid) [inline assets/sass/patterns/molecules/pull-quote.scss]: Just removes a magic value. Well, 14 is still a magic value...

File: assets/sass/patterns/molecules/pull-quote.scss [hunk 2]
```diff
@@ -2,21 +2,16 @@
...
 
   p {
-    // TODO: Refactor pullquote-typeg to set appropriate line-height as well as font-size
     @include pullquote-typeg();
     line-height: 1.2;
     margin: 0;
     padding: 0;
-
-    @media only all and (min-width: #{$bkpt-pull-quote}px) {
-      @include pullquote-typeg(true);
```
Comment (thewilkybarkid) [inline assets/sass/patterns/molecules/pull-quote.scss]: Doesn't make a difference.

File: source/_patterns/01-molecules/components/pull-quote.yaml [hunk 3]
```diff
@@ -12,7 +12,5 @@ schema:
     cite:
       type: string
       minLength: 1
-    asParagraph:
-      type: boolean
```
Comment (thewilkybarkid) [inline source/_patterns/01-molecules/components/pull-quote.yaml]: Don't understand why this was there?

File: assets/sass/patterns/molecules/article-section.scss [hunk 4]
```diff
@@ -143,26 +143,6 @@ $arrow-size: 5;
...
-  }
-
-  blockquote cite {
-    color: $color-text-secondary;
-    font-family: $font-primary;
-    @include font-size(14);
-    font-style: italic;
-    line-height: 1.3;
-  }
-
```
Comment (thewilkybarkid) [inline assets/sass/patterns/molecules/article-section.scss]: Don't understand why pull quotes inside sections were treated differently?

File: assets/sass/patterns/molecules/pull-quote.scss (hunk 1 above)
Comment (thewilkybarkid) [reply assets/sass/patterns/molecules/pull-quote.scss]: screen shot 2017-09-07 at 08 26 30

The gap before a pull quote is bigger, is that meant? (The quote mark is there, but looks a bit odd to me.)

Comment (davidcmoulton) [reply assets/sass/patterns/molecules/pull-quote.scss]: This has been through pre-launch design checks. The important thing is to make sure the content falls back onto the baseline grid after the quote.

File: source/_patterns/01-molecules/components/pull-quote.yaml (hunk 3 above)
Comment (thewilkybarkid) [reply source/_patterns/01-molecules/components/pull-quote.yaml]: Yep, they are.

**Pull request merged**